src/index.html

```diff
@@ -30,6 +31,7 @@
 		and useful generating open knowledge for all.">
 	<meta property="og:site_name" content="Loklak Search - Distributed Open Source Search for Twitter and Social Media with Peer to Peer Technology">
 
+  <noscript><a>Please enable Javacript.</a></noscript>
```

Comment:
I think this can go without the anchor tags (`<a>...</a>`).

Reply:
ok @singhpratyush , will remove it 👍